src/App.js

```diff
@@ -14,18 +14,21 @@ class App extends Component {
       platforms: [],
       genresToFilter: [],
       platformsToFilter: [],
-      multiplayer: [null, true, false]
+      filteredGames: [],
+      multiplayer: [null, true, false],
      }
   }
   componentDidMount = () => {
     fetch('http://whateverly-datasets.herokuapp.com/api/v1/games1811')
       .then(response => response.json())
       .then(games => {
         this.setState({
-          games: games.games1811
+          games: games.games1811,
+          filteredGames: games.games1811
```

Comment:
App.js line 27, what's the explanation for having both a 'games' property AND a 'filteredGames' property in state? Couldn't we just filter through the 'games' property whenever we need to in order to avoid redundancy?

Reply:
I think I understand why we have the two in **_state_**: one is for all of the games. While the filteredGames is for after we have called a function to filter though the (all)games array, saving them to this second (filteredGames) array.
- However, it looks like at the point of fetch we are assigning all games to both arrays, is that correct?

Reply:
- @DekayHaHa  I know that you are currently having the issue where we cannot search specific titles once we have filtered genre/platform:
I think that we'll have to introduce some more conditionals in our filter functions, so that when those functions are called, they are not only searching the games array, but the filtered array too. Maybe if the filteredGames array is empty, we then look to the whole games array to apply the filter.

Reply:
Jake's comment is pretty on point -- you want to avoid storing duplicate data as much as possible. Filtering in place when you need to render games, based on whatever is in your `genresToFilter` and `platformsToFilter` is a more accepted pattern here. As you build more complex applications, you'll realize how easy it is for duplicate data to get out of sync and how many bugs it can cause down the road. No worries if this is too difficult to refactor at this point, we can discuss it more as a class during our project retro/evals.